chain/proofs/mock/mock.go

```diff
@@ -0,0 +1,125 @@
+package mock
```

Comment:
this is a minimised version of ./storage/sealer/mock/mock.go that has just what we need